src/store/tasks/slice.ts

```diff
@@ -0,0 +1,25 @@
+import * as taskActions from './actions';
+import { ITaskState } from '../../types/tasks';
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState: ITaskState = {
+  label: '',
+  status: 'Default',
+  modal: false,
+  addingTask: false,
+};
+
+export const taskSlice = createSlice({
```

Comment:
You need to create tasks slice where you'll store all tasks from here: 
![image](https://github.com/VladyslavGilyuk/chi-pet-project/assets/68470050/c3c17935-ab06-42f1-a4fc-a530eeb04935)
You will have `tasks` in state and several actions like addTask, editTask, deleteTask etc. 
 it is totally incorrect slice without any sense.

Reply:
To do it correctly please watch any video on youTube about creation ToDo application with React + RTK

Reply:
fix: rearanged with RTK